### PR TITLE
[ETHEREUM-CONTRACTS] backport 1.4.1 fixes

### PR DESCRIPTION
**Pull request by hellwolf**
- deployment scripts improvements
- [SECURITY] CFA: check flow sender instead of msg sender in order to cover ACL use

Co-authored-by: 0xdavinchee <0xdavinchee@gmail.com>

### CODE REVIEW

File: packages/ethereum-contracts/test/contracts/utils/CFAv1Forwarder.test.js [hunk 1]
```diff
@@ -0,0 +1,657 @@
+import {SignerWithAddress} from "@nomiclabs/hardhat-ethers/signers";
```
Comment (hellwolf) [inline packages/ethereum-contracts/test/contracts/utils/CFAv1Forwarder.test.js]: stray file, terminate!

Comment (hellwolf) [reply packages/ethereum-contracts/test/contracts/utils/CFAv1Forwarder.test.js]: oh, no it's not stray, it's just JS

File: packages/js-sdk/package.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{
```
Comment (hellwolf) [inline packages/js-sdk/package.json]: this file needs to go

**Comment by hellwolf** [comment]
Seems good let's merge!

**Review by 0xdavinchee — APPROVED** [review]
Looks good - LGTM

**Pull request merged**